CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v5.0.0
```

Comment:
So it's a major release without anything to change for us poor consumers? You're doing this just for people who might have extended the component, correct?

Reply:
yes, BC break unfortunately :/

Reply:
Not only you can extend the component, but can inject a component class into another component by `ViewChild`

Reply:
Very true indeed, thus the breaking change. Makes perfect sense now, thanks.

Reply:
Hidden bitcoin miner will land in v6 :+1: